### PR TITLE
Pyverbs fixes

### PR DESCRIPTION
**Pull request by jgunthorpe**
Various fixes to the use of cython and the pyverbs test runner.

### CODE REVIEW

File: buildlib/pyverbs_functions.cmake [hunk 1]
```diff
@@ -9,6 +9,7 @@ function(rdma_cython_module PY_MODULE)
     include_directories(${PYTHON_INCLUDE_DIRS})
     add_custom_command(
       OUTPUT "${CFILE}"
+      MAIN_DEPENDENCY "${PYX}"
```
Comment (noaos) [inline buildlib/pyverbs_functions.cmake]: What does this change? I mean, cython will fail if you don't provide pyx files anyway.

Comment (noaos) [reply buildlib/pyverbs_functions.cmake]: I usually delete the build/ directory before I call ninja, so I wasn't aware.

Comment (jgunthorpe) [reply buildlib/pyverbs_functions.cmake]: Don't do things like that, it hides defects like this that effect the rest of us.

**Comment by rleon** [comment]
Jason,
Can you please rebase so we will merge it?

**Pull request merged**